subsys/usb/class/netusb/Kconfig

```diff
@@ -0,0 +1,77 @@
+# Kconfig - netusb configuration options
+
+#
+# Copyright (c) 2017 Intel Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menu "USB Device Networking support"
+	depends on USB_DEVICE_STACK
+
+config USB_DEVICE_NETWORK
```

Comment:
what is the idea behind this kconfig option? it has no description and is automatically selected

Reply:
@nashif idea here is that we implement some other functions (RNDIS, etc). Common code is built whenever network function is enabled. I have working solution with several functions where it does make sense